source/pkg/states/PlayState.hx

```diff
@@ -0,0 +1,13 @@
+package;
+
+import flixel.FlxSprite;
+import flixel.FlxState;
+
+class PlayState extends FlxState
+{
+	override public function create() {}
+
+	override public function update(elapsed:Float) {}
```

Comment:
A super statement in here "super.update(elapsed:Float);" Very important, allows objects we've added to this state to update